src/test/kotlin/com/ampnet/identityservice/controller/VeriffControllerTest.kt

```diff
@@ -0,0 +1,250 @@
+            val payload = verificationService.generatePayload(ADDRESS.toString())
+            testContext.signedPayload = "0x" + KEY_PAIR.signMessage(payload.toByteArray()).toHex()
+        }
+        suppose("User has an account") {
+            databaseCleanerService.deleteAllUsers()
+            databaseCleanerService.deleteAllUserInfos()
+            databaseCleanerService.deleteAllVeriffSessions()
+            testContext.user = createUser()
+            val veriffSession = VeriffSession(
+                "44927492-8799-406e-8076-933bc9164ebc",
+                testContext.user.address,
+                "https://alchemy.veriff.com/v/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...",
+                testContext.user.address,
+                "https://alchemy.veriff.com/",
+                "created",
+                false,
+                ZonedDateTime.now(),
+                VeriffSessionState.SUBMITTED
+            )
+            testContext.veriffSession = veriffSessionRepository.save(veriffSession)
+        }
+        suppose("Veriff posted declined decision") {
+            databaseCleanerService.deleteAllVeriffDecisions()
+            val decision = VeriffDecision(
+                testContext.veriffSession.id,
+                VeriffStatus.declined,
+                9102,
+                "Physical document not used",
+                101,
+                "2020-12-04T10:45:37.907Z",
+                "2020-12-04T10:45:31.000Z",
+                ZonedDateTime.now()
+            )
+            veriffDecisionRepository.save(decision)
+        }
+        suppose("Veriff will return new session") {
+            val response = getResourceAsText("/veriff/response-new-session.json")
+            mockVeriffResponse(response, HttpMethod.POST, "/v1/sessions/")
+        }
+
+        verify("Controller will return new veriff session") {
+            val request = objectMapper.writeValueAsString(VeriffRequest(testContext.signedPayload))
+            val result = mockMvc.perform(
+                post("/veriff/session")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(request)
+            )
+                .andExpect(status().isOk)
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andReturn()
+            val veriffResponse: ServiceVerificationResponse = objectMapper.readValue(result.response.contentAsString)
+            assertThat(veriffResponse.decision).isNotNull
+            assertThat(veriffResponse.verificationUrl)
+                .isEqualTo("https://alchemy.veriff.com/v/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.new-url")
+        }
+    }
+
+    @Test
+    fun mustStoreUserInfoFromVeriff() {
+        suppose("User has no user info") {
+            databaseCleanerService.deleteAllUsers()
+            databaseCleanerService.deleteAllUserInfos()
+            testContext.user = createUser("5750f893-29fa-4910-8304-62f834338f47")
```

Comment:
```suggestion
            testContext.user = createUser("0x0472ec0185ebb8202f3d4ddb0226998889663cf2 ")
```
Use eth address instead of UUID.